app/src/main/res/values/styles.xml

```diff
@@ -1,7 +1,7 @@
 <resources>
 
     <!-- Base application theme. -->
-    <style name="AppTheme" parent="Theme.AppCompat.Light.DarkActionBar">
+    <style name="AppTheme" parent="Theme.MaterialComponents.Light.DarkActionBar.Bridge">
```

Comment:
Are we getting trouble with our support for older devices? we wanted to support down to minSdk version 14.

Reply:
Afaik no, Android X is the new AppCompat, so. The Bridge theme is just in place until all. AppCompat widgets have been replaced with the corresponding AndroidX widgets.

Reply:
@stefan-niedermann for details, see https://developer.android.com/jetpack/androidx/